packages/openlmis/src/Adaptor.js

```diff
@@ -0,0 +1,130 @@
+import { expandReferences } from '@openfn/language-common/util';
+import { execute as commonExecute } from '@openfn/language-common';
+import * as util from './Utils';
+
+/**
+ * State object
+ * @typedef {Object} OpenLMISState
+ * @private
+ * @property data - the parsed response body
+ * @property response - the response from the HTTP server, including headers, statusCode, body, etc
+ * @property references - an array of all previous data objects used in the Job
+ **/
+
+/**
+ * Options provided to the HTTP request
+ * @typedef {Object} RequestOptions
+ * @property {object} body - body data to append to the request.
+ * @property {object} query - An object of query parameters to be encoded into the URL.
+ * @property {object} headers - An object of headers to append to the request.
+ */
+
+/**
+ * Execute a sequence of operations.
+ * Wraps `language-common/execute`, and prepends initial state for odk.
```

Comment:
:exclamation: This mentions ODK

Can we search for any `odk` strings in `packages/openlmis`?